src/content/docs/en/guides/content-collections.mdx

```diff
@@ -61,11 +61,12 @@ If you find yourself working with different types of content, you should create
   - **newsletter/** 
     - week-1.md
     - week-2.md
-    - week-3.md
-  - **authors/** split different content types into new collections
-    - grace-hopper.md
-    - alan-turing.md
-    - batman.md
+  - **blog/**
+    - post-1.md
+    - post-2.md
+  - **authors/**
+    - grace-hopper.json
+    - alan-turing.json
```

Comment:
Originally had two collections here, one content and one data. But I realized this _might_ make it confusing what "different types of content" means; does "type" mean content vs. data or something about the structure? Three collections should hopefully make that clearer!